extensions/chghost-3.2.md

```diff
@@ -16,20 +16,37 @@ The chghost client capability allows servers to directly inform clients about a
 host or user change without having to send fake quits or joins. This capability
 MUST be referred to as `chghost` at capability negotiation time.
 
-When enabled, clients will get the `CHGHOST` message to designate the host of a
-user changing for clients on common channels with them.
+When the capability is enabled, servers MUST send the `CHGHOST` message to
+clients sharing the same channel as the target client, informing those clients
+that the user or host of the target client has changed.
+
+Servers SHOULD also send the `CHGHOST` command to the client who changed their
+host, as well as the previously-unspecified `396` numeric, whose exact value
```

Comment:
We can't... specify that implementers send a numeric without being clear about the syntax of that numeric. And especially if that numeric already has conflicting definitions between existing vendors. For what it's worth, [info](https://defs.ircdocs.horse/defs/numerics.html#rpl-visiblehost-396).

If we want to allow servers to keep sending their `RPL_VISIBLEHOST`/`RPL_YOURDISPLAYEDHOST `/`RPL_HOSTHIDDEN` numerics we should just not mention it at all and leave it up to the server, especially since newer implementations shouldn't need to pick between existing conflicting numeric definitions.

Reply:
That's fair, this was a complicated thing I found incredibly hard to word anyways. I'll remove it tomorrow.

Reply:
We could maybe mention how CHGHOST differs from the numeric (e.g. 396 doesn't consistently report changes to ident) and say that servers MAY continue to send it alongside CHGHOST without mandating it.

Reply:
I want to avoid servers going "oh I can just replace 396 with CHGHOST now" because some clients might still be relying on the numeric.